wp-content/themes/core/blocks/tribe/terms/render.php

```diff
@@ -15,45 +15,35 @@
 
 echo '<div ' .  wp_kses_data( get_block_wrapper_attributes() ) . '>';
 
-if ( 0 === count( $terms_block_terms ) ) {
-	echo '<!-- Terms block: No terms to list. -->';
```

Comment:
We render the block wrapper div regardless, so I don't see the comment as necessary.